### DISCUSSION TITLE
Why does Oxidized shows the dm when clicking over a person?

### DISCUSSION THREAD

**Discussion by MarcAnt01** [BreeceW/Oxidized#94] [General]
I actually expect to open the same dialog, with roles etc. like it happens on Discord apps.
Futhermore since I cannot dm with Oxidized and I haven't had any conversation with most of the people there and I don't think it actually makes sense, afaik when I click on a person mention, it's because I don't know them, so seeing the propic, roles etc. gives me an idea

**Comment by BreeceW** [comment]
The current behavior is in place until #55 is implemented.

**Reply by MarcAnt01** [reply]
I didn't even remember having made that ticket
